app.py

```diff
@@ -0,0 +1,14 @@
+# import the required packages
```

Comment:
we no longer need this comment, plus all others, the code should be self-documenting